autopush/endpoint.py

```diff
@@ -174,7 +175,7 @@ def chid(self):
     @chid.setter
     def chid(self, value):
         """Set the ChannelID and record to _client_info"""
-        self._chid = value
+        self._chid = normalize_id(value)
         self._client_info["channelID"] = value
```

Comment:
I'm confused, this will normalize the CHID whenever it's set. that should produce consistent log entries, no?